itinerary-library/scripts/build_presets.py

```
@@ -1,534 +1,17 @@
-zip_path=Path('/mnt/data/roamwise-itinerary-library-v1.0.0.zip')
-if zip_path.exists(): zip_path.unlink()
-with zipfile.ZipFile(zip_path,'w',zipfile.ZIP_DEFLATED,compresslevel=7) as z:
-    for p in OUT.rglob('*'):
-        if p.is_file(): z.write(p,p.relative_to(OUT.parent))
-
-print(json.dumps({'destinations':len(C),'variants_each':4,'preset_html':len(htmls),'pdfs':len(pdfs),'zip':str(zip_path),'zip_mb':round(zip_path.stat().st_size/1024/1024,2)},indent=2))
+root=Path(__file__).parent
+catalog=root/'catalog.zlib.b64'
+if not catalog.exists():
+    chunks=sorted(root.glob('catalog.part*.txt'))
+    if not chunks: raise SystemExit('Missing catalog source fragments')
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)